lib/metadata/schema-parser.js

```diff
@@ -45,23 +51,27 @@ function SchemaParser(cc) {
   this.selectUdt = null;
   this.selectAggregates = null;
   this.selectFunctions = null;
+  this.selectVirtualTable = null;
+  this.selectVirtualColumns = null;
 }
 
 /**
  * @param name
  * @param durableWrites
  * @param strategy
  * @param strategyOptions
+ * @param virtual
  * @returns {{name, durableWrites, strategy, strategyOptions, tokenToReplica, udts, tables, functions, aggregates}|null}
  * @protected
  */
-SchemaParser.prototype._createKeyspace = function (name, durableWrites, strategy, strategyOptions) {
+SchemaParser.prototype._createKeyspace = function (name, durableWrites, strategy, strategyOptions, virtual) {
   const ksInfo = {
     name: name,
     durableWrites: durableWrites,
     strategy: strategy,
     strategyOptions: strategyOptions,
     tokenToReplica: null,
+    virtual: virtual === true ? true : false,
```

Comment:
Minor nit: can be simplified to `virtual: virtual === true` or `virtual: !!virtual`